Uploaded the initial commit

lbrlekovi20 opened the pull request:
Uploaded the IoT asset data simulator and Mosquitto MQTT broker.

lbrlekovi20 committed:
Uploaded the initial commit

vdraganic21 reviewed (changes requested):
- Simulator runs.
- Please remove comments from code and do some refactoring in next commits.
- Fix bug with config loading.
 
After that's done I'd merge with dev branch. :)

lbrlekovi20 committed:
The simulator properly finds and reads config file, removed unnecessary comments.

vdraganic21 committed:
Added broker startup messages. Made changes to env file loading.
Deleted config.env as it will be created by the program itself if it doesnt exist.

vdraganic21 reviewed (approved):
Everything works now. I redid the env loading and added some more startup messages.

I'd say it's ready to merge.

mmathis21 reviewed (approved):
Issues have been fixed.

Ready to merge.
